app/templates/components/modals/tax-info-modal.hbs

```diff
@@ -44,32 +44,30 @@
         </label>
       </div>
     </div>
-    {{#if tax.shouldSendInvoice}}
```

Comment:
The option for organiser whether he wants to send taxed invoice or not  . 
@shreyanshdwivedi How are you planning to manage this, Since all of them are optional fields

Reply:
I think nothing will be affected due to this. I'm still unable to get you point
@uds5501 @mrsaicharan1 maybe you guys can explain me what @kushthedude means

Reply:
@shreyanshdwivedi maybe remove the fields out of the `tax.shouldSendInvoice` condition but still provide this check box? Because we do need this boolean condition for sending invoice. Otherwise your implementation is fine

Reply:
@uds5501 @kushthedude I haven't removed the checkbox as you can see on line 37. Then what's the problem?

Reply:
@shreyanshdwivedi my bad, didn't see that there, implementation's alright then!